library/alloc/src/string.rs

```diff
@@ -2106,7 +2106,8 @@ impl_eq! { Cow<'a, str>, &'b str }
 impl_eq! { Cow<'a, str>, String }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl Default for String {
+#[rustc_const_unstable(feature = "const_default_impls", issue = "none")]
```

Comment:
Are these shown in rustdoc? Not sure if there's much point in adding these without, it would be very hard to find.

Reply:
These attributes are required to gate the const impl. I will look at how rustdoc deals with this and possibly open a separate PR

Reply:
Also the RFC for `impl const` is not even accepted, so I don't know if we should change rustdoc for this. What do you think @jyn514? [tracking issue](67792)

> This RFC has not yet been accepted. It is being implemented on a provisional basis to [evaluate the potential fallout](https://github.com/rust-lang/rfcs/pull/2632#issuecomment-567699174).

Reply:
If the RFC hasn't been accepted, I definitely don't think we should add it to the standard library.